source/tutorials/monitor-minio-tenant.rst

```diff
@@ -62,6 +62,20 @@ The command returns a bearer token similar to the following:
      static_configs:
      - targets: ['HOSTNAME:9000']
 
+If using a self signed certificate, add the following to skip verification.
```

Comment:
@kerneltime just to make sure we're clear here - this is if the *MinIO* Tenant is using self-signed certificates (i.e. auto-cert)

Reply:
Correct @ravindk89